taccsite_cms/static/site_cms/css/src/site.css

```diff
@@ -4,16 +4,15 @@
 /* SEE: https://confluence.tacc.utexas.edu/x/IAA9Cw */
 
 /* SETTINGS */
-/* Shared with Portal */
 @import url("_imports/settings/border.css");
-/* Unique to CMS */
-@import url("_imports/settings/color.css");
-@import url("_imports/settings/props.max-width.css");
-@import url("_imports/settings/props.space.css");
-@import url("_imports/settings/font.css");
+/* Unique to CMS */    @import url("./_imports/settings/color.css");
+@import url("_imports/settings/max-width.css");
+/* Unique to CMS */    @import url("./_imports/settings/space.css");
+/* Unique to CMS */    @import url("./_imports/settings/font.css");
```

Comment:
**Note.** I did not re-order these, _only_ so that local testing—comparing build output—is easier. I would like to, because the order is actually irrelevant to the build, but important for legibility.

Reply:
I can see that. You want to make that a post-merge task? Or do it now?

Reply:
Ugh. I want to do it now, but that'll make my test notes inaccurate, so I'll do it in a tiny post-merge PR.

Reply:
K, legibility improved in follow-up PR merged) [#484](https://github.com/TACC/Core-CMS/pull/484/files).